### DISCUSSION TITLE
V2raya GFWList not updating

### DISCUSSION THREAD

**Discussion by epeqs** [v2rayA/v2rayA#1475] [General]
Hey guys, I set up v2raya today in OpenWrt but somehow the GFWList won't update -> it always brings the error: Get "[https://raw.githubusercontent.com/v2rayA/dist-v2ray-rules-dat/202408062211/geosite.dat 1](https://raw.githubusercontent.com/v2rayA/dist-v2ray-rules-dat/202408062211/geosite.dat)": tls: failed to verify certificate: x509: certificate is not valid for any names, but wanted to match [raw.githubusercontent.com](http://raw.githubusercontent.com/)

**Comment by guoguo12343** [comment]
手动更新这个文件即可geosite.dat，放在/usr/share/v2ray下面重命名为LoyalsoldierSite.dat，在/usr/share/xray中建立软连接。
